packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -32139,7 +32139,7 @@ export default function UpdateOrgForm(props) {
     const queryData = async () => {
       const record = idProp ? await DataStore.query(Org, idProp) : orgModelProp;
       setOrgRecord(record);
-      const linkedComments = record ? await record.comments.toArray() : [];
+      const linkedComments = record ? await record?.comments?.toArray() : [];
```

Comment:
This might return undefined which we don't want. Can you rewrite to 
```
const linkedComments = record && await record.comments?.toArray() || [];
```

Reply:
Thanks for the catch, I've gone through and updated this to coalesce to the empty array.